cmd/config/config.go

```diff
@@ -23,15 +23,15 @@ func main() {
 
 	var in, out, keyStr string
 	var inplace bool
-	var version int
+	var version uint
 
 	fs := flag.NewFlagSet("config", flag.ExitOnError)
 	fs.Usage = func() { usage(fs) }
 	fs.StringVar(&in, "in", defaultCfgFile, "The config input file to process")
 	fs.StringVar(&out, "out", "[in].out", "The config output file")
 	fs.BoolVar(&inplace, "edit", false, "Edit; Save result to the original file")
 	fs.StringVar(&keyStr, "key", "", "The key to use for AES encryption")
-	fs.IntVar(&version, "version", 0, "The version to downgrade to")
+	fs.UintVar(&version, "version", 0, "The version to downgrade to")
```

Comment:
A truly petty nit, this version is limited to `16,383` versions and `LatestVersion` is limited to `65,535`  versions. Claustrophobic!

I suppose we have 16,380 versions' time to update this to uin64Var is needed 😄

Reply:
I was thinking WTF is arm/Apple silicon really this cooked? But looks like you got that value because the version you inputted was > MaxUint16 and we just type cast it without checking. Without that, should fit 65355 fine without any issues.

Reply:
@thrasher- is correct: The issue is that that we're not checking for overflow on uint16.
That's actually relatively bad because a fat finger error could cause a downgrade to an actual existing version.

So: We have 65534 available versions.

Fixed https://github.com/gbjk/gocryptotrader/commit/e23a2f1173

Slight niggle in that the version check for overflow happens inside the cmd wrapper so has a different format.
If someone puts in 65536 and then corrects to 65530 they'll get a second error about what the highest version  available *actually* is.
However I'm pretty comfortable with limited convenience in don't-shoot-your-foot-off circumstances.

Reply:
Can't wait for v65420 now that this is resolved